realm/realm-library/src/main/cpp/io_realm_mongodb_sync_Sync.cpp

```diff
@@ -72,15 +73,6 @@ JNIEXPORT void JNICALL Java_io_realm_mongodb_sync_Sync_nativeReconnect(JNIEnv* e
     CATCH_STD()
 }
 
-JNIEXPORT void JNICALL Java_io_realm_mongodb_sync_Sync_nativeCreateSession(JNIEnv* env, jclass, jlong j_native_config_ptr)
-{
-    try {
-        auto& config = *reinterpret_cast<Realm::Config*>(j_native_config_ptr);
-        _impl::RealmCoordinator::get_coordinator(config)->create_session(config);
```

Comment:
where is the session created now?

Reply:
It is created automatically by Object Store when you open the Realm.